pkg/service/roomservice.go

```diff
@@ -158,9 +152,15 @@ func (s *RoomService) DeleteRoom(ctx context.Context, req *livekit.DeleteRoomReq
 		return nil, twirpAuthError(err)
 	}
 
-	_, err := s.roomClient.DeleteRoom(ctx, s.topicFormatter.RoomTopic(ctx, livekit.RoomName(req.Room)), req)
-	if !errors.Is(err, psrpc.ErrNoResponse) {
-		return &livekit.DeleteRoomResponse{}, err
+	done, err := s.startRoom(ctx, livekit.RoomName(req.Room))
```

Comment:
One concern about this: in case the room doesn't exist.. this would effectively create it.. instead of returning 404.

should we try to load it from Store first?